perf/roles/trex-ndr/defaults/main.yml

```diff
@@ -6,3 +6,4 @@ profile: vlan
 vlan: 212
 expected_pps: 1
 outer_left_mac: de:ed:de:ed:00:01
+queue_ratios: 1.0:1.0:1.0:1.0
```

Comment:
No need for floats here. It makes it harder to read. Probably best with `1:1:1:1`.